src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Form, Col, Row, Input, Button, Select } from "antd";
+import { RoleConstantArray } from "../../../constants/RoleConstants";
+import { AccountStatusConstantsArray } from "../../../constants/AccountStatusConstants";
+
+const SearchAccountForm = ({ resetAction, ...antdProps }) => {
```

Comment:
is this worth to put in separate file like this ? or only need to put this function in the same table container file ?

Reply:
because the table container file is too long